src/app.js

```diff
@@ -118,6 +118,8 @@ class SAFEApp extends EventEmitter {
                 let newPath;
                 if (!path || !path.length) {
                   newPath = '/index.html';
+                } else if (path === '/') {
+                  newPath = 'index.html';
```

Comment:
`else if (path[path.length - 1] === '/') {` condition is for accessing subdirectory `index.html` files, where `else if (path === '/') {` is for accessing `index.html` on rootDir